lms/djangoapps/courseware/tests/test_module_render.py

```diff
@@ -2,21 +2,22 @@
 """
 Test for lms courseware app, module render unit
 """
-from functools import partial
+import ddt
+import itertools
 import json
 
 from bson import ObjectId
-import ddt
 from django.http import Http404, HttpResponse
 from django.core.urlresolvers import reverse
 from django.conf import settings
 from django.test.client import RequestFactory
 from django.test.utils import override_settings
 from django.contrib.auth.models import AnonymousUser
+from functools import partial
```

Comment:
This is super-nit territory, but since you moved this import: usually, the sections of imports are: stdlib, third-party libs, project files.  Not sure why functools moved into the third-party lib section.